packages/skeleton-react/src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from "react";
+import { ProgressProps } from "./types";
+
+export default function Progress({
+	value,
+	min = 0,
+	max = 100,
+	ariaLabelledby = "",
+	// Root
+	base = "overflow-x-hidden",
+	bg = "bg-surface-200-800",
+	height = "h-2",
+	rounded = "rounded",
+	classes = "",
+	// Meter
+	meterBase = "h-full",
+	meterBg = "bg-primary-500",
+	meterRounded = "rounded",
+	meterTransition = "transition-[width]",
+	meterIndeterminateAnim = "indeterminate",
+	meterClasses = "",
+}: ProgressProps) {
+	useEffect(() => {
+		if (min > max) {
+			console.error(
+				`The minimum value: "${min}" is greater than the maximum value: "${max}"`,
+			);
+		}
+	}, [min, max]);
+
+	const indeterminate = value === undefined;
+	const fillPercentage = indeterminate ? 50 : ((value! - min) / (max - min)) * 100;
+	const width = `${fillPercentage}%`;
+
+	const rxIndeterminate = indeterminate ? meterIndeterminateAnim : "";
+
+	return (
+		<>
+			<div
+				role="progressbar"
+				aria-labelledby={ariaLabelledby}
+				aria-valuenow={value}
+				aria-valuemin={min}
+				aria-valuemax={max}
+				className={`${base} ${bg} ${height} ${rounded} ${classes}`}
+			>
+				<div
```

Comment:
I think the styled div is more than adequate here, but MDN has an interesting take. Using an SVG!
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role#examples

The benefit here is we could enable more complex visual styling. Such as using a Stroke Linecap like the Radial Progress bars:
https://www.skeleton.dev/components/progress-radials#styling
(change the option from `butt` at the bottom-right of the preview)

I'd only suggest this if you're feeling like an extra challenge!

Reply:
This definitely seems interesting, I'll take a look once the components are pretty much done